undertow/src/main/java/org/wildfly/extension/undertow/logging/UndertowLogger.java

```diff
@@ -442,4 +442,8 @@ public interface UndertowLogger extends BasicLogger {
     @Message(id = 110, value = "The use of security realms at runtime is unsupported.")
     OperationFailedException runtimeSecurityRealmUnsupported();
 
+    @LogMessage(level = WARN)
+    @Message(id = 111, value = "Annotation: '%s' cannot be used on Servlet: '%s'")
```

Comment:
This should say something about what will happen, e.g. the annotation will have no effect (if that's true.)

Reply:
Thanks.